src/NLog.Extensions.Logging/Config/NLogLoggingConfiguration.cs

```diff
@@ -272,28 +263,26 @@ private bool IsTargetsSection()
             /// <summary>
             /// Target-config inside Wrapper-Target
             /// </summary>
-            /// <param name="child"></param>
-            /// <returns></returns>
             private bool IsTargetWithinWrapper(IConfigurationSection child)
             {
                 return _nameOverride == TargetKey && GetConfigKey(child).EqualsOrdinalIgnoreCase(TargetKey) && child.GetChildren().Count() == 1;
             }
 
             private IEnumerable<LoggingConfigurationElement> GetTargetsDefaultConfigElements()
             {
-                if (DefaultTargetWrapperSection != null)
-                    yield return new LoggingConfigurationElement(DefaultTargetWrapperSection, true, DefaultWrapper);
+                if (TargetDefaultWrapperSection != null)
+                    yield return new LoggingConfigurationElement(TargetDefaultWrapperSection, DefaultWrapper);
 
-                if (DefaultTargetParametersSection != null)
+                if (TargetDefaultParametersSection != null)
                 {
-                    foreach (var targetParameters in DefaultTargetParametersSection.GetChildren())
+                    foreach (var targetParameters in TargetDefaultParametersSection.GetChildren())
                     {
-                        yield return new LoggingConfigurationElement(targetParameters, true, DefaultTargetParameters);
+                        yield return new LoggingConfigurationElement(targetParameters, DefaultTargetParameters);
                     }
                 }
             }
 
-            private bool AlreadyReadChild(IConfigurationSection child, IConfigurationSection variables, IConfigurationSection defaultWrapper, IConfigurationSection defaultTargetParameters)
+            private bool AlreadyReadChild(IConfigurationSection child, IConfigurationSection variables, IConfigurationSection targetDefaultWrapper, IConfigurationSection targetDefaultParameters)
```

Comment:
Isn't defaultTargetParameters more in line with:

✔️ DO choose easily readable identifier names.

For example, a property named HorizontalAlignment is more English-readable than AlignmentHorizontal.

From https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions

Reply:
Well I like that first word specifies the area of functionality. And "default" is very generic and gives no information. Just like I prefer `FilterDefaultAction` over `DefaultFilterAction`.

Reply:
DefaultFilterAction is also more in line with english and the microsoft guideline.